### PR TITLE
test: add periodic to periodic suite names

### PR DESCRIPTION
**Pull request by ellistarn**
Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml [hunk 1]
```diff
@@ -58,26 +58,26 @@ data:
      apiVersion: tekton.dev/v1beta1
      kind: PipelineRun
      metadata:
-       generateName: "upgrade-suite"
+       generateName: "upgrade-periodic-"
```
Comment (spring1843) [inline test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml:61]: Do we need the trailing `-`?

File: test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml [hunk 2]
```diff
@@ -58,26 +58,26 @@ data:
...
        serviceAccountName: karpenter-tests
     EOF
-    echo "CREATED PIPELINE RUN for upgrade-suite"
+    echo "CREATED PIPELINE RUN for upgrade-periodic"
     cat <<EOF | kubectl create -f -
      apiVersion: tekton.dev/v1beta1
      kind: PipelineRun
      metadata:
-       generateName: "ipv6-suite"
+       generateName: "ipv6-periodic-"
```
Comment (spring1843) [inline test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml:73]: similar trailing extra `-`

**Review by bwagner5 — APPROVED** [review]
mostly LGTM

**Review by spring1843 — APPROVED** [review]

File: test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml (hunk 2 above)
Comment (ellistarn) [reply test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml:73]: If you look at the CI tests, these are missing a `-`. This addition is intentional.

Comment (ellistarn) [reply test/infrastructure/clusters/test-infra/karpenter-tests/pipeline-trigger-cron.yaml:73]: image

**Pull request merged**